charts/sdp-prototype/templates/sdp-devices-deploy.yml

```diff
@@ -74,20 +74,39 @@ spec:
         env:
         - name: TANGO_HOST
           value: databaseds-tango-base-{{ .Release.Name }}:10000
-        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_master:0.2.1
+        image: nexus.engageska-portugal.pt/sdp-prototype/tangods_sdp_master:0.2.3
         imagePullPolicy: Always
```

Comment:
If you put the named version, what's the need for Always as pull policy

Reply:
These new images have been tested by the NCRA team and have confirmed that it is working. 
I am not sure what other ways we could test it.

Reply:
I'll talk with them. Never mind. Thanks.

Reply:
sure will do.